pkg/trace/writer/trace.go

```diff
@@ -6,20 +6,20 @@
 package writer
 
 import (
+	"compress/gzip"
 	"errors"
 	"runtime"
 	"strings"
 	"sync"
 	"time"
 
-	"github.com/DataDog/zstd"
-
 	pb "github.com/DataDog/datadog-agent/pkg/proto/pbgo/trace"
 	"github.com/DataDog/datadog-agent/pkg/trace/config"
 	"github.com/DataDog/datadog-agent/pkg/trace/info"
 	"github.com/DataDog/datadog-agent/pkg/trace/log"
 	"github.com/DataDog/datadog-agent/pkg/trace/telemetry"
 	"github.com/DataDog/datadog-agent/pkg/trace/timing"
+	"github.com/DataDog/zstd"
```

Comment:
Nit:
I think this one needs to go down with `"github.com/DataDog/datadog-go/v5/statsd"` since it's from a different repository.

Reply:
Fixed thanks a lot.